dev/io.openliberty.microprofile.telemetry.2.0.logging.internal/src/io/openliberty/microprofile/telemetry20/logging/internal/MpTelemetryLogMappingUtils.java

```diff
@@ -35,6 +37,8 @@
 @Trivial
 public class MpTelemetryLogMappingUtils {
 
+    private static final TraceComponent tc = Tr.register(OpenTelemetryLogHandler.class, "TELEMETRY", "io.openliberty.microprofile.telemetry.internal.common.resources.MPTelemetry");
```

Comment:
The register class does not match the class name.

Reply:
Fixed!